src/lib/mina_base/test/zkapp_command/transaction_commitments.ml

```diff
@@ -0,0 +1,39 @@
+open Core_kernel
+open Mina_base.Zkapp_command
+open Mina_base
+open Snarky_backendless
+open Snark_params.Tick
+
+let%test_module "get_transaction_commitments" =
+  ( module struct
+    let tree_gen =
+      let open Quickcheck.Let_syntax in
+      let%map account_gen = Account_update.gen_with_events_and_actions in
+      { Call_forest.Tree.account_update = account_gen
+      ; account_update_digest = ()
+      ; calls =
+          ([] : ( (Account_update.t, unit, unit) Call_forest.tree
+                , unit )
+                With_stack_hash.Stable.V1.t
+                list)
+      }
+
+    let zkapp_type_gen =
```

Comment:
Maybe you could reuse here the generator defined in `src/lib/mina_base/test/zkapp_command/valid_size.ml` and just discard the `length`.

Reply:
I'll do this. Thanks.